### PR TITLE
Filter schema failure known issues in upgrade_test

### PR DESCRIPTION
**Pull request by fruch**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- ~~[ ] New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- ~~[ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (`hydra unit-tests`)
- ~~[ ] I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by fruch** [comment]
@amoskong 

still need your confirmation that this error line can be ignored or not:
```
2019-07-31 12:40:40.329: (DatabaseLogEvent Severity.CRITICAL): type=DATABASE_ERROR regex=Exception  line_number=72360 node=Node rolling-upgrade-2019-1-n-ubuntu-xen-db-node-808c9296-0-2 [34.74.63.169 | 10.142.0.131] (seed: False)
2019-07-31T12:40:30+00:00  rolling-upgrade-2019-1-n-ubuntu-xen-db-node-808c9296-0-2 !WARNING | scylla:  [shard 0] storage_service - Failed to pull schema from 10.142.0.143: exceptions::mutation_write_failure_exception (Operation failed for system.local - received 0 responses and 1 failures from 1 CL=ONE.)
```

**Comment by amoskong** [comment]
Hi @fruch `Failed to pull schema` should be ignored.

**Comment by amoskong** [comment]
BTW, can we make the filter list configurable?

**Comment by fruch** [comment]
@amoskong what do you mean by configurable ? that it would be part of the test yaml ? and can be overwritten by environment variable ?

**Comment by amoskong** [comment]
Yes.

**Comment by fruch** [comment]
@amoskong you asked for it in the configuration, you got it :)

would be easily used in more tests if needed

**Comment by fruch** [comment]
@bentsi,
1) amos asked for this option, i.e. this way you can overwrite it if needed, or share that logic for more then one test, since it's not part of the test flow directly, it's not that critical where it's written down.

2) amos check, that those error are acceptable during upgrades, and there login in the test the count how many of them are in the log.

File: unit_tests/test_events.py [hunk 1]
```diff
@@ -132,6 +132,23 @@ def test_filter_repair(self):
         self.assertIn("not filtered", log_content_after)
         self.assertNotIn("repair id 1", log_content_after)
 
+    def test_filter_upgrade(self):
```
Comment (bentsi) [inline unit_tests/test_events.py]: approved the PR but not sure why do we need this test, it looks the same as `test_filter_repair`

**Pull request merged**